src/gamepad/input-type.vala

```diff
@@ -0,0 +1,11 @@
+// This file is part of GNOME Games. License: GPLv3
+
+public enum LibGamepad.InputType {
+	AXIS,
+	BUTTON,
+	INVALID;
+
+	public static uint length () {
```

Comment:
I doesn't seem to be used to me, if it's not we better remove it, otherwise tell me so I don't annoy you with it later.